src/core/services/layout/layout.scss

```diff
@@ -19,71 +19,90 @@
   display: flex;
 }
 
-.layout-column {
+.layout-column, .md-css-only [layout=column] {
   flex-direction: column;
   -webkit-flex-direction: column;
 }
 
-.layout-row {
+.layout-row, .md-css-only [layout=row] {
   flex-direction: row;
   -webkit-flex-direction: row;
 }
 
 .layout-padding > .flex-sm,
-.layout-padding > .flex-lt-md
+.layout-padding > .flex-lt-md,
+.md-css-only [layout-padding] > [flex-sm],
+.md-css-only [layout-padding] > [flex-lt-md]
 {
   padding: $layout-gutter-width / 4;
 }
+
 .layout-padding,
 .layout-padding > .flex,
 .layout-padding > .flex-gt-sm,
 .layout-padding > .flex-md,
-.layout-padding > .flex-lt-lg
+.layout-padding > .flex-lt-lg,
+.md-css-only [layout-padding],
+.md-css-only [layout-padding] > .md-css-only [flex],
+.md-css-only [layout-padding] > .md-css-only [flex-gt-sm],
+.md-css-only [layout-padding] > .md-css-only [flex-md],
+.md-css-only [layout-padding] > .md-css-only [flex-lt-lg]
 {
   padding: $layout-gutter-width / 2;
 }
 .layout-padding > .flex-gt-md,
-.layout-padding > .flex-lg
+.layout-padding > .flex-lg,
+.md-css-only [layout-padding] > .md-css-only [flex-gt-md],
+.md-css-only [layout-padding] > .md-css-only [flex-lg]
 {
   padding: $layout-gutter-width / 1;
 }
 
-
 .layout-margin > .flex-sm,
-.layout-margin > .flex-lt-md
+.layout-margin > .flex-lt-md,
+.md-css-only [layout-margin] >  .md-css-only [flex-sm],
+.md-css-only [layout-margin] >  .md-css-only [flex-lt-md]
 {
   margin: $layout-gutter-width / 4;
 }
+
 .layout-margin,
 .layout-margin > .flex,
 .layout-margin > .flex-gt-sm,
 .layout-margin > .flex-md,
-.layout-margin > .flex-lt-lg {
+.layout-margin > .flex-lt-lg,
+.md-css-only [layout-margin],
+.md-css-only [layout-margin] >  .md-css-only [flex],
+.md-css-only [layout-margin] >  .md-css-only [flex-gt-sm],
+.md-css-only [layout-margin] >  .md-css-only [flex-md],
+.md-css-only [layout-margin] >  .md-css-only [flex-lt-lg]
+{
   margin: $layout-gutter-width / 2;
 }
+
 .layout-margin > .flex-gt-md,
-.layout-margin > .flex-lg
+.layout-margin > .flex-lg,
+.md-css-only [layout-margin] >  .md-css-only [flex-gt-md],
+.md-css-only [layout-margin] >  .md-css-only [flex-lg]
 {
   margin: $layout-gutter-width / 1;
 }
 
-
-
-.layout-wrap {
+.layout-wrap, .md-css-only [layout-wrap]  {
   flex-wrap: wrap;
 }
 
-.layout-fill {
+.layout-fill, .md-css-only [layout-fill] {
   margin: 0;
   min-height: 100%;
   width: 100%;
 }
 @-moz-document url-prefix() {
-  .layout-fill {
+  .layout-fill, .md-css-only [layout-fill] {
     margin: 0;
     width: 100%;
-    min-height: auto;
-    height: inherit;
+    min-height: 100%;
+    height: 100%;
```

Comment:
I don't think it needs a comment, but can you briefly explain why these were changed from auto/inherit to 100%?